BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -5,12 +5,36 @@ import Foundation
 /// An enum to represent a feature flag sent by the server
 ///
 enum FeatureFlag: String, Codable {
+    /// Flag to enable/disable email verification during registration
+    /// This flag introduces a new flow for account creation
+    case emailVerification = "email-verification"
+
     /// A feature flag for the intro carousel flow.
     case nativeCarouselFlow = "native-carousel-flow"
 
     /// A feature flag for showing the unassigned items banner.
     case unassignedItemsBanner = "unassigned-items-banner"
-    /// Flag to enable/disable email verification during registration
-    /// This flag introduces a new flow for account creation
-    case emailVerification = "email-verification"
+
+    // MARK: Test Flags
+
+    /// A test feature flag that isn't remotely configured.
```

Comment:
```suggestion
    /// A test feature flag that isn't locally configured.
```

Reply:
I think I have a slight preference to "isn't remotely configured" since we don't have a mechanism for locally configuring feature flags.